### PR TITLE
add MapsWithMe package name to AndroidManifest.xml (fix #11134)

### LINKED ISSUE #11134: maps.me navigation not available
Reported by some beta users, could not test it yet.

With the last version I was able to navigate with "MAPS.ME". This does no longer work.

### PR DESCRIPTION
**Pull request by moving-bits**
## Description
Adds MapsWithMe package names to `AndroidManifest.xml` to be able to query those packages even when running on Android 11

### CODE REVIEW

**Comment by moving-bits** [comment]
fixed first mwm package name and added osmand

File: main/AndroidManifest.xml [hunk 1]
```diff
@@ -947,6 +949,7 @@
         <package android:name="googoo.android.btgps" />
         <package android:name="menion.android.locus" />
         <package android:name="menion.android.whereyougo" />
+        <package android:name="net.osmand" />
```
Comment (Lineflyer) [inline main/AndroidManifest.xml:952]: net.osmand.plus missing

Comment (moving-bits) [reply main/AndroidManifest.xml:952]: hmpf.

added it :-)

Comment (Lineflyer) [reply main/AndroidManifest.xml:952]: Does this support some placeholders or regex patterns? I guess not...

Comment (moving-bits) [reply main/AndroidManifest.xml:952]: not that I know of

and no string placeholders as well, sadly - would be nice, as we have many of those package names already defined in `strings_not_translatable.xml`

**Pull request merged**